framework/src/Volo.Abp.EventBus.Boxes/Volo/Abp/EventBus/Boxes/InboxProcessor.cs

```diff
@@ -116,14 +120,14 @@ await DistributedEventBus
                 else
                 {
                     Logger.LogDebug("Could not obtain the distributed lock: " + DistributedLockName);
-                    await TaskDelayHelper.DelayAsync(15000, StoppingToken); //TODO: Config?
+                    await TaskDelayHelper.DelayAsync(EventBusBoxesOptions.DelayTimeSpan.Milliseconds, StoppingToken);
                 }
             }
         }
 
         protected virtual async Task DeleteOldEventsAsync()
         {
-            if (LastCleanTime != null && LastCleanTime > Clock.Now.AddHours(6)) //TODO: Config?
+            if (LastCleanTime != null && LastCleanTime > Clock.Now.Add(EventBusBoxesOptions.CleanOldEventTimeIntervalSpan))
```

Comment:
It seems `CleanOldEventTimeIntervalSpan` is a positive value and you are adding it to DateTime.Now. In this case, how `LastCleanTime` can be greater than this value. I suppose my implementation also was wrong.
`if (LastCleanTime != null && LastCleanTime.AddHours(6) > Clock.Now)` seems true. Right?